Track Your Reps/Views/FeedTableCell.swift

```diff
@@ -13,7 +13,7 @@ class FeedTableCell: UITableViewCell {
     
     var event: Event? {
         didSet {
-            eventQuestionLabel.text = event?.question
+            eventQuestionLabel.text = "\(event!.isBill ? "I'm a bill! - " : "")\(event?.question)"
```

Comment:
just an example of using bill data in the feed table

`.isBill` is an easy computed var helper to determine whether an event is/has a bill